ft_bzero.c

```diff
@@ -0,0 +1,26 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_bzero.c                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ldamiens <ldamiens@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2021/11/03 16:38:28 by ldamiens          #+#    #+#             */
+/*   Updated: 2021/11/12 11:51:07 by ldamiens         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <stddef.h>
+
+// On peut utiliser le ft_memset
+void	ft_bzero(void *string, size_t n)
```

Comment:
Prefer using short names for values 
```suggestion
void	ft_bzero(void *str, size_t n)
```